doc/source/WorkingWithThePythonScriptingAPI/TutorialNodeGraphEditingInPython/index.md

```diff
@@ -101,32 +101,32 @@ mySphere['radius']
 > Caution :
 > Because Python dictionaries do not have built-in overwrite protection, you can accidentally and irrecoverably replace nodes and plugs with assignments that use existing node names, like `root['Sphere'] = ...`. Use dictionary syntax with care.
 
-Just like with nodes, you can insert a reference to a plug by dragging. Try inserting a reference to radius plug of the Sphere node:
+Just like with nodes, you can insert a reference to a plug by dragging. Try inserting a reference to Radius plug of the Sphere node:
  
-1. Select the Sphere node in the _Graph Editor_.
-2. Click and drag the **label** of the radius plug from the _Node Editor_ (the cursor will change to ![](images/plug.png "a plug")).
-3. Release it onto the input field of the _Python Editor_.
+1. Select the Sphere node in the Graph Editor.
+2. Click and drag the **label** of the Radius plug from the Node Editor (the cursor will change to ![](images/plug.png "a plug")).
+3. Release it onto the input field of the Python Editor.
 
 A reference to `root['Sphere']['radius']` will be inserted. This is identical to `mySphere['radius']` from earlier. Notice how when you drag and drop plugs, the reference is formatted in dictionary syntax.
 
 > Important :
-> Dragging and dropping plugs is a core technique when using the _Python Editor_. It can speed up your node graph editing and inspecting considerably.
+> Dragging and dropping plugs is a core technique when using the Python Editor. It can speed up your node graph editing and inspecting considerably.
 
 
 ## Retrieving a plug value ##
 
-The `getValue()` method retrieves a plug's value. Try it on the `Cs` (colour) plug of the OpenGLShader node:
+The `getValue()` method retrieves a plug's value. Try it on the `Cs` (Color) plug of the OpenGLShader node:
```

Comment:
I'm not sure `Color` should be capitalised in this case, it's just saying "btw, thats the plug that changes the color". Its name is `Cs` (plug and label).